config/tables.php

```diff
@@ -93,6 +93,7 @@
     'buttons' => [
         'global' => [
             'create' => [
+                'name' => 'create',
```

Comment:
do we really need this? we could use the key to determine the name. If not, this will be a breaking change.